build/tools/before_install.sh

```diff
@@ -42,6 +42,23 @@ case $(uname -s) in
             done
 
             $SUDO apt-get install -y $libtoolkit_dev $pkg_install
+
+            if [ "$wxUSE_ASAN" = 1 ]; then
+                # Enable the `-dbgsym` repositories.
+                echo "deb http://ddebs.ubuntu.com $(lsb_release -cs) main restricted universe multiverse
```

Comment:
It is better to use long options in the scripts, especially if they're not well-known (e.g. `rm -f` is probably fine). So could you please do
```suggestion
                echo "deb http://ddebs.ubuntu.com $(lsb_release --codename --short) main restricted universe multiverse
```
and the same below (alternatively define a variable `$codename` once and use it)?